paper.md

```diff
@@ -103,11 +103,29 @@ on sparse/banded matrices have a memory requirement growing linearly with the
 number of grid points. In addition, the structure of the operators can be taken
 into account for operator-vector multiplications, usually resulting in speed-ups
 of an order of magnitude or more on consumer hardware. For example, the application
-of the sixth-order (in the interior) finite difference SBP operator on a grid
-with 1000 nodes takes roughly 330 ns on a consumer CPU from 2017 (Intel® Core™ i7-8700K)
-using version v0.5.5 of SummationByPartsOperators.jl. In contrast, the same
-operation takes roughly 3.9 microseconds using a sparse matrix format used in
-other implementations of this operator [@almquist2017optimized].
+an optimized the sixth-order (in the interior) finite difference SBP operator
+[@almquist2017optimized] on a grid with 1000 nodes takes roughly 330 ns
+on a consumer CPU from 2017 (Intel® Core™ i7-8700K) using version v0.5.5 of
+SummationByPartsOperators.jl. In contrast, the same operation takes roughly
+3.9 microseconds using a sparse matrix format used in other implementations of
+this operator [@almquist2017optimized]. This benchmark is based on the following
+code, which also provides a very basic example of SummationByPartsOperators.jl.
+```julia
+julia> using Pkg; Pkg.add("SummationByPartsOperators") # if not installed previously
```

Comment:
Would it maybe be better to just state the code, not the code in REPL? Sure that also leaves out the results, but maybe one can answer in a sentence what the last command yields?

That way the example could easier be read I think.

Reply:
I removed the REPL code

Reply:
That looks better I think. We could check how the one multiline renders and maybe put that in one line or put the closing bracket in its own line – but that's merely a typesetting thing.

Reply:
See below (https://github.com/ranocha/SummationByPartsOperators.jl/pull/109#discussion_r672467668)

Reply:
this is fine now.